modules/ocf/manifests/firewall/post.pp

```diff
@@ -68,4 +69,59 @@
       before => undef,
     }
   }
+
+  # drop from internal zone exceptions: tsunami, werewolves, death, and dev- versions
+  # hard code the addresses in case of DNS malfunction
+
+  $drop_all = ['tsunami', 'werewolves', 'death', 'dev-tsunami', 'dev-werewolves', 'dev-death']
+
+  $drop_all.each |String $s| {
+    ocf::firewall::firewall46 { "997 drop internal zone exception, (${s})":
+      opts  => {
+        chain  => 'PUPPET-INPUT',
+        proto  => ['tcp', 'udp'],
+        action => 'drop',
+        source => $s
+      }
+    }
+  }
+
+  firewall {
+    '998 allow from internal zone (IPv4)':
+      chain     => 'PUPPET-INPUT',
+      src_range => '169.229.226.5-169.229.226.90',
+      proto     => ['tcp', 'udp'],
+      action    => 'accept';
+
+    '998 allow ssh from desktops (IPv6)':
```

Comment:
Consider modifying `ocf::firewall::allow_ssh` to take address ranges and using that instead.